decidim-comments/spec/commands/create_comment_spec.rb

```diff
@@ -70,49 +70,16 @@ module Comments
             end.to change(Comment, :count).by(1)
           end
 
-          it "sends a notification to the corresponding users except the comment's author" do
-            follower = create(:user, organization: organization)
-
-            expect(commentable)
-              .to receive(:users_to_notify_on_comment_created)
-              .and_return([follower, author])
-
-            expect_any_instance_of(Decidim::Comments::Comment) # rubocop:disable RSpec/AnyInstance
-              .to receive(:id).at_least(:once).and_return 1
-
-            expect_any_instance_of(Decidim::Comments::Comment) # rubocop:disable RSpec/AnyInstance
-              .to receive(:root_commentable).at_least(:once).and_return commentable
-
-            expect(Decidim::EventsManager)
-              .to receive(:publish)
-              .with(
-                event: "decidim.events.comments.comment_created",
-                event_class: Decidim::Comments::CommentCreatedEvent,
-                resource: commentable,
-                recipient_ids: [follower.id],
-                extra: {
-                  comment_id: a_kind_of(Integer)
-                }
-              )
+          it "sends the notifications" do
+            creator_double = double(create: true)
```

Comment:
It's better to use `instance_double(NewCommentNotificationCreator, create: true)` so this test would fail if `NewCommentNotificationCreator` changes its public methods.